portal-web/test/functional/com/liferay/portalweb/tests/enduser/publications/PublicationsPage.testcase

```diff
@@ -25,6 +25,91 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for LPS-177151 - Add style book and apply it within Publications"
+	@priority = 3
+	test CanAddStyleBookAndApplyItWithinPublications {
+		task ("Create Publication and Site") {
+			JSONPublications.addPublication(publicationName = "Test Publication");
+
+			JSONPublications.selectPublication(publicationName = "Test Publication");
+
+			Refresh();
+
+			JSONGroup.addGroup(groupName = "Test Site");
+
+			Refresh();
+		}
+
+		ApplicationsMenu.gotoSite(site = "Test Site");
+
+		task ("Create Style Book with times as Font Family Base") {
+			StyleBooksAdmin.openStyleBooksAdmin(siteURLKey = "test-site");
+
+			StyleBooksAdmin.addStyleBooks(styleBookName = "Test Style Book Name");
+
+			StyleBooks.selectForm(formName = "Typography");
+
+			Type(
+				key_fieldLabel = "Font Family Base",
+				key_groupLabel = "Font Family",
+				locator1 = "TextInput#GROUPED_TEXT_INPUT",
+				value1 = "times");
+
+			StyleBooks.clickManagementBar();
+
+			StyleBooks.waitForAutoSave();
+
+			StyleBooks.clickPublish();
+		}
+
+		task ("Create Content Page and Select Test Style Book Name") {
+			PagesAdmin.openPagesAdmin(siteURLKey = "test-site");
+
+			ContentPages.addPage(pageName = "Example Page");
+
+			PageEditor.gotoTab(tabName = "Page Design Options");
+
+			Navigator.gotoNavTab(navTab = "Style Book");
+
+			StyleBooks.selectStyleBookViaPageDesignOptions(styleBookName = "Test Style Book Name");
+
+			PageEditor.publish();
+		}
+
+		task ("View the Font Family Base is times") {
+			StyleBooksAdmin.openStyleBooksAdmin(siteURLKey = "test-site");
+
+			StyleBooksAdmin.gotoStyleBookEllipsisItem(
+				item = "Edit",
+				styleBookName = "Test Style Book Name");
+
+			StyleBooks.selectForm(formName = "Typography");
+
+			AssertTextEquals(
+				key_fieldLabel = "Font Family Base",
+				key_groupLabel = "Font Family",
+				locator1 = "TextInput#GROUPED_TEXT_INPUT",
+				value1 = "times");
+		}
```

Comment:
Out of scope for this test. We only need to assert that the font family is applied to Example Page.